python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -927,6 +952,15 @@ def dispose(self) -> None:
     # run storage
     @public
     def get_run_by_id(self, run_id: str) -> Optional[DagsterRun]:
+        """Get a run by id.
```

Comment:
I think it's probably worth deep linking the `DagsterRun` object here, and mentioning `run_id` by name; otherwise can imagine some potential small-scale confusion